.gitignore

```diff
@@ -13,10 +13,10 @@ debian/*
 !debian/rules
 !debian/source
 
+build/
```

Comment:
This line is unnecessary. :) 

https://mesonbuild.com/FAQ.html#how-do-i-ignore-the-build-directory-in-my-vcs

Reply:
Oh, that's nice. Thanks!